packs/actions/dampening-harmonics.json

```
@@ -0,0 +1,33 @@
+{
+    "_id": "cczvLO5r7QuHaXwx",
+    "img": "systems/pf2e/icons/actions/TwoActions.webp",
+    "name": "Dampening Harmonics",
+    "system": {
+        "actionType": {
+            "value": "action"
+        },
+        "actions": {
+            "value": 2
+        },
+        "category": "offensive",
+        "description": {
+            "value": "<p><strong>Frequency</strong> once per hour</p>\n<hr />\n<p><strong>Effect</strong> You establish a force field that grants you resistance 10 to damage dealt by spells and magical abilities with the trait of your magiphage ability, except for force damage. The force field lasts for 10 minutes. Each time the field prevents damage, the duration decreases by 1 minute.</p>"
+        },
+        "frequency": {
+            "max": 1,
+            "per": "PT1H"
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Howl of The Wild"
```

Should "the" be lower case?
```suggestion
            "title": "Pathfinder Howl of the Wild"
```